release_test_compare.py

```diff
@@ -65,6 +65,10 @@ def parse_args():
                         help='Main RAWDATA folder with results from previous version.')
     parser.add_argument('target_folder',
                         help='Testing RAWDATA folder to compare with.')
+    parser.add_argument('--workers',
+                        default=50,
```

Comment:
Instead of hard coding the default, make a guess based on available memory and processors. We do something similar in the [docker wrapper](https://github.com/cfe-lab/MiCall/blob/932e55e1d737b77af2dc4b1ad043d909ad70e9e0/micall_docker.py#L1166-L1170).

Reply:
Done!